x-pack/plugin/esql/compute/src/test/java/org/elasticsearch/compute/aggregation/SumLongAggregatorFunctionTests.java

```diff
@@ -47,22 +47,6 @@ public void assertSimpleOutput(List<Block> input, Block result) {
         assertThat(((LongBlock) result).getLong(0), equalTo(sum));
     }
 
-    public void testOverflowFails() {
```

Comment:
TODO: Instead test for null?

Reply:
++. I'd love to have a proper test for that.